lxd/instance/drivers/driver_qemu.go

```diff
@@ -1449,12 +1511,13 @@ fi
 cp udev/99-lxd-agent.rules /lib/udev/rules.d/
 cp systemd/lxd-agent.service /lib/systemd/system/
 cp systemd/lxd-agent-9p.service /lib/systemd/system/
+cp systemd/lxd-agent-virtiofs.service /lib/systemd/system/
 systemctl daemon-reload
-systemctl enable lxd-agent.service lxd-agent-9p.service
+systemctl enable lxd-agent.service lxd-agent-9p.service lxd-agent-virtiofs.service
 
 echo ""
 echo "LXD agent has been installed, reboot to confirm setup."
-echo "To start it now, unmount this filesystem and run: systemctl start lxd-agent-9p lxd-agent"
+echo "To start it now, unmount this filesystem and run: systemctl start lxd-agent"
```

Comment:
Does that actually trigger both of the needed units?

Reply:
The way I understand systemd, it should trigger the dependencies.